config/locales/cy.yml

```diff
@@ -34,6 +34,36 @@ cy:
         see_all:
           text: "Gweld ein holl hystadegau"
           path: "/government/statistics?departments[]=%{organisation}"
+      news_and_communications:
+        title: "News and communications"
+        see_all:
+          text: "See all news and communications"
+          path: "/news-and-communications?organisations[]=%{organisation}"
```

Comment:
Still waiting on translations and correct links for the translation files.

Reply:
You don't need to add these placeholder translations. If an entry doesn't exist in the locale, then the entry for "en" is used as the default.